projects/laji/src/app/+project-form/submissions/statistics/common/accepted-document-approval/accepted-document-approval.component.html

```diff
@@ -29,6 +29,6 @@
           (click)="acceptNamedPlaceChanges()"
           class="btn btn-sm btn-success ml-2">Hyväksy muutokset</button>
 </div>
-<div class="col-xs-12" [style.height.px]="1000" [style.display]="isAdmin && geometriesDiff ? 'block' : 'none'">
+<div *ngIf="isAdmin" class="col-xs-12" [style.height.px]="1000">
```

Comment:
Just to make sure, is it intentional that this element is also shown when `geometriesDiff` is `undefined`?